conf/webpack/webpack.config.js

```diff
@@ -3,58 +3,73 @@
  * All rights reserved
  * mailto:info AT sonarsource DOT com
  */
-const path = require('path');
-const autoprefixer = require('autoprefixer');
-
-const autoprefixerOptions = {
-  browsers: [
-    'last 3 Chrome versions',
-    'last 3 Firefox versions',
-    'Safari >= 8',
-    'Edge >= 12',
-    'IE 11'
-  ]
-};
-
-const output = path.join(__dirname, '../../target/classes/static');
+const path = require("path");
+const autoprefixer = require("autoprefixer");
 
 module.exports = {
+  // Define the entry points here. They MUST have the same name as the page_id
+  // defined in src/main/java/org/sonarsource/plugins/example/web/MyPluginPageDefinition.java
   entry: {
-    'measures_history': ['./src/main/js/app-measures_history.js'],
-    'sanity_check': ['./src/main/js/app-sanity_check.js'],
+    // Using Vanilla JS:
+    global_page: ["./src/main/js/global_page/index.js"],
+
+    // Using Backbone JS:
+    project_page: ["./src/main/js/project_page/index.js"],
+
+    // Using React:
+    portfolio_page: ["./src/main/js/portfolio_page/index.js"],
+    admin_page: ["./src/main/js/admin_page/index.js"]
   },
   output: {
-    path: output,
-    filename: '[name].js'
+    // The entry point files MUST be shipped inside the final JAR's static/
+    // directory.
+    path: path.join(__dirname, "../../target/classes/static"),
+    filename: "[name].js"
   },
   resolve: {
-    root: path.join(__dirname, 'src/main/js')
+    root: path.join(__dirname, "src/main/js")
   },
   externals: {
-    lodash: '_',
-    react: 'React',
-    'react-dom': 'ReactDOM',
-    'react-redux': 'ReactRedux',
-    'react-router': 'ReactRouter',
-    'sonar-request': 'SonarRequest',
-    'sonar-measures': 'SonarMeasures',
-    'sonar-components': 'SonarComponents'
+    // React 16.8 ships with SonarQube, and should be re-used to avoid 
+    // collisions at runtime.
+    react: "React",
+    "react-dom": "ReactDOM",
+    // Register the Sonar* globals as packages, to simplify importing.
+    // See src/main/js/common/api.js for more information on what is exposed 
+    // in SonarRequest.
+    "sonar-request": "SonarRequest",
+    // TODO: provide an example
+    "sonar-measures": "SonarMeasures",
+    // See src/main/js/portfolio_page/components/MeasuresHistory.js for some
+    // examples using React components from SonarQube.
+    "sonar-components": "SonarComponents"
```

Comment:
Like I said, not a fan of documenting sonar-measures and sonar-components.

Reply:
And I'm a bit shocked to see that in fact they were in the previous example... We have never been aware of this in the frontend team...

Reply:
Yes, exactly 😅 . Which is why I thought, why not? Note that this documents the 7.x series, _not_ 8.x. We can remove this and add info about `sonar-ui-common` when it's ready.

Reply:
Meaning, the new link on `sonar-docs` specifically links to the `7.x` branch on this repo, not on master. And the `README.md` specifies 7.9 LTS.

Reply:
The thing is, I don't really feel like documenting what we have. It's going to be long and tedious. Might as well let them see the code, and deal with it. 

That being said, I will update the links to make it point to the `branch-7.9` branch instead of master on `sonar-enterprise`.